docs/reference/modules/node.asciidoc

```diff
@@ -89,93 +91,85 @@ memory and CPU in order to deal with the gather phase.
 ===============================================
 
 [[master-node]]
-==== Master-eligible node
+==== Manager node
 
-The master node is responsible for lightweight cluster-wide actions such as
+The manager node is responsible for lightweight cluster-wide actions such as
 creating or deleting an index, tracking which nodes are part of the cluster,
 and deciding which shards to allocate to which nodes. It is important for
-cluster health to have a stable master node.
+cluster health to have a stable manager node.
 
-Any master-eligible node that is not a <<voting-only-node,voting-only node>> may
-be elected to become the master node by the <<modules-discovery,master election
-process>>.
+Any control node that is not a <<voting-only-node,voting-only node>> can
+become the manager through the <<modules-discovery, election process>>.
 
-IMPORTANT: Master nodes must have access to the `data/` directory (just like
+IMPORTANT: Manager nodes must have access to the `data/` directory (just like
```

Comment:
```suggestion
IMPORTANT: Control nodes must have access to the `data/` directory (just like
```

Reply:
> I would prefer to use consistent terminology for the election process where it's necessary to mention it (which isn't often) but we could definitely consider changing that to something other than "election". In particular I like "selection", that fits the manager metaphor better IMO without completely losing the idea of votes and quorums.

I agree, there really isn't a need to explicitly call it an election.